fullstack/src/components/ui/tooltip.jsx

```diff
@@ -0,0 +1,26 @@
+"use client"
+
+import * as React from "react"
+import * as TooltipPrimitive from "@radix-ui/react-tooltip"
+
+import { cn } from "@/lib/utils"
+
+const TooltipProvider = TooltipPrimitive.Provider
+
+const Tooltip = TooltipPrimitive.Root
+
+const TooltipTrigger = TooltipPrimitive.Trigger
+
+const TooltipContent = React.forwardRef(({ className, sideOffset = 4, ...props }, ref) => (
+  <TooltipPrimitive.Content
+    ref={ref}
+    sideOffset={sideOffset}
+    className={cn(
```

Comment:
Can we format this so it's a bit nicer to read? Maybe run it through Prettier?

Reply:
Yep it's all just a prebuilt component so hadn't even looked at it!